### PR TITLE
fix rspec 3.3

### PR DESCRIPTION
**Pull request by michaelglass**
split this off from #27

### CODE REVIEW

File: lib/rspec_ext/rspec_ext.rb [hunk 1]
```diff
@@ -11,10 +11,16 @@ def clear_exception
 module RSpec
   module Core
     class ExampleGroup
+      def clear_memoized
+        # __memoized is private method and is defined in rspec 3.3
+        __memoized.instance_variable_get(:@memoized).clear
+      rescue NameError
```
Comment (michaelglass) [inline lib/rspec_ext/rspec_ext.rb]: don't use `rescue` for control flow

**Comment by michaelglass** [comment]
@eitoball can you take a look at my mild changes?

File: lib/rspec_ext/rspec_ext.rb [hunk 2]
```diff
@@ -8,13 +8,22 @@ def clear_exception
   end
 end
 
+require 'pry'
```
Comment (eitoball) [inline lib/rspec_ext/rspec_ext.rb]: Should remove this line.

**Comment by eitoball** [comment]
Almost LGTM, see https://github.com/y310/rspec-retry/pull/32#discussion_r34314589

I intentionally didn't use `__init_memoize`, but I now think that using it is much better. Thanks!

File: lib/rspec_ext/rspec_ext.rb (hunk 2 above)
Comment (michaelglass) [inline lib/rspec_ext/rspec_ext.rb]: this is why we code review thanks.

**Pull request merged**